griptape/structures/structure.py

```diff
@@ -21,7 +22,7 @@
 
 
 @define
-class Structure(ABC, RuleMixin, SerializableMixin):
+class Structure(RuleMixin, SerializableMixin, RunnableMixin, ABC):
```

Comment:
Can you make `ABC` the first one again? (all style)

Reply:
Unfortunately not. Because `RunnableMixin` implements ABC we get this from pyright:
```
Cannot create consistent method ordering
```